nevergrad/optimization/optimizerlib.py

```diff
@@ -1389,26 +1389,57 @@ def __init__(self, optimizers: Sequence[Union[base.OptimizerFamily, Type[base.Op
         super().__init__()
 
 
+chainDEwithR = Chaining([RandomSearch, DE], ["num_workers"]).with_name("chainDEwithR", register=True)
+chainDEwithRsqrt = Chaining([RandomSearch, DE], ["sqrt"]).with_name("chainDEwithRsqrt", register=True)
+chainDEwithRdim = Chaining([RandomSearch, DE], ["dimension"]).with_name("chainDEwithRdim", register=True)
+chainDEwithR30 = Chaining([RandomSearch, DE], [30]).with_name("chainDEwithR30", register=True) 
 chainDEwithLHS = Chaining([LHSSearch, DE], ["num_workers"]).with_name("chainDEwithLHS", register=True)
 chainDEwithLHSsqrt = Chaining([LHSSearch, DE], ["sqrt"]).with_name("chainDEwithLHSsqrt", register=True)
 chainDEwithLHSdim = Chaining([LHSSearch, DE], ["dimension"]).with_name("chainDEwithLHSdim", register=True)
 chainDEwithLHS30 = Chaining([LHSSearch, DE], [30]).with_name("chainDEwithLHS30", register=True)
-
+chainDEwithMetaRecentering = Chaining([MetaRecentering, DE], ["num_workers"]).with_name("chainDEwithMetaRecentering", register=True)
```

Comment:
I'm thinking of moving all these instantiations in some separate files, this way they can still be defined somewhere, but they do not make the main optimizers namespace full of variants. We'll discuss this in January